src/components/ExpandableCard.tsx

```diff
@@ -54,96 +52,40 @@ const ExpandableCard = ({
-            }}
-            _expanded={{
-              backgroundColor: "transparent",
-            }}
+            className="w-full p-6 transition-colors hover:bg-gray-50 hover:text-black md:p-6 dark:hover:bg-gray-800 dark:hover:text-white [&[data-state=open]]:bg-transparent [&[data-state=open]]:text-black dark:[&[data-state=open]]:text-white"
           >
-            <Box
-              display="flex"
-              width="100%"
-              alignItems="center"
-              flexDir={{ base: "column", sm: "row" }}
-              textAlign="start"
-            >
-              <Box marginBottom={{ base: 2, sm: 0 }} me={{ base: 0, sm: 4 }}>
-                <Box
-                  display="flex"
-                  alignItems="center"
-                  sx={{
-                    svg: { me: "1.5rem" },
-                  }}
-                  my="4"
-                >
-                  {!!Svg && <Svg />}
-                  <Text fontSize="xl" fontWeight="semibold" flex="1" m="0">
-                    {title}
-                  </Text>
-                </Box>
-                <Text
-                  fontSize="sm"
-                  color="text200"
-                  marginBottom="0"
-                  width="fit-content"
-                >
+            <Flex className="w-full flex-col items-center text-left sm:flex-row">
+              <VStack className="items-center">
+                <HStack className="mb-2 mt-4">
+                  {Svg && <Svg className="mr-6" />}
+                  <h3 className="text-xl font-semibold">{title}</h3>
+                </HStack>
+                <p className="mb-0 w-fit text-sm text-gray-600">
                   {contentPreview}
-                </Text>
-              </Box>
-              <Text
-                fontSize="md"
-                color="primary.base"
-                ms={{ base: undefined, sm: "auto" }}
-                mt="auto"
-                mb="auto"
-              >
+                </p>
+              </VStack>
+              <span className="my-auto text-md text-purple-600 sm:ml-auto dark:text-purple-400">
                 {t(isVisible ? "less" : "more")}
-              </Text>
-            </Box>
-          </AccordionButton>
-        </Heading>
-        <AccordionPanel
-          p="0"
-          mt="0"
-          paddingX="6"
-          paddingBottom="6"
-          paddingTop="0"
-          onClick={onClick}
-        >
-          <Box
-            fontSize="md"
-            color="text"
-            paddingTop="6"
-            borderTop="1px solid"
-            borderColor="border"
-          >
-            {children}
-          </Box>
-        </AccordionPanel>
-      </AccordionItem>
-    </Accordion>
+              </span>
+            </Flex>
+          </AccordionTrigger>
+          <AccordionContent className="cursor-pointer p-6 pt-0 md:p-6 md:pt-0">
+            <div className="border-t border-gray-200 pt-6 text-md text-gray-800 dark:border-gray-600 dark:text-white">
```

Comment:
Lets go with our default color on border and body color.
```suggestion
            <div className="border-t pt-6 text-md text-body">
```